forge-gui/res/cardsfolder/upcoming/magar_of_the_magic_strings.txt

```diff
@@ -0,0 +1,11 @@
+Name:Magar of the Magic Strings
+ManaCost:1 B R
+Types:Legendary Creature Minotaur Performer
+PT:3/3
+A:AB$ ChangeZone | Cost$ 1 B R | ValidTgts$ Instant.YouOwn,Sorcery.YouOwn | TgtPrompt$ Select target instant or sorcery card in your graveyard | Origin$ Graveyard | Destination$ Battlefield | RememberLKI$ True | FaceDown$ True | FaceDownPower$ 3 | FaceDownToughness$ 3 | FaceDownSetType$ Creature | AnimateSubAbility$ DBAnimate | StackDescription$ REP Note_{p:You} notes & target instant or sorcery card in your graveyard_{c:Targeted} & put_puts | SpellDescription$ Note the name of target instant or sorcery card in your graveyard and put it onto the battlefield face down. It's a 3/3 creature with "Whenever this creature deals combat damage to a player, you may create a copy of the card with the noted name. You may cast the copy without paying its mana cost" and "If this creature would leave the battlefield, exile it instead of putting it anywhere else."
+SVar:DBAnimate:DB$ Animate | RememberObjects$ RememberedLKI | Defined$ Remembered | Triggers$ DealsCDTrig | Replacements$ ReplaceDies | Duration$ Permanent
+SVar:DealsCDTrig:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigPlay | TriggerZones$ Battlefield | TriggerDescription$ Whenever this creature deals combat damage to a player, you may create a copy of the card with the noted name. You may cast the copy without paying its mana cost.
+SVar:TrigPlay:DB$ Play | CopyFromRemembered$ True | WithoutManaCost$ True | Optional$ True
```

Comment:
- You should still use Imprinted too so the RememberObjects$ only grabs one card?
- Why add any extra code when CopyCard$ already exists...
- Also ValidSA$ missing
- Cleanup missing

Reply:
RememberObjects does find / remember one card... probably a bug because of things being turned face-down? I checked it in the debugger

Similar issue with the Cleanup... nothing is remembered on Magar anyway. Another / same bug?
<img width="1025" alt="image" src="https://github.com/Card-Forge/forge/assets/103371817/bcc2fd8d-9df6-4289-8ebe-308d1a01febf">

`CopyCard$` was flipping the face-down card and putting the copy on the battlefield instead of casting it, hence the reasoning for the new param

Reply:
`Defined$ RememberedLKI | ZoneRegardless$ True | CopyCard$ True` works fine for me?
what steps are needed for error?